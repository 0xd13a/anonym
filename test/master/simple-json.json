{"a": {"b": "Norma Fisher", "e": "f"}, "x": "7c65c1e5-82e2-4662-b728-b4fa42485e3a", "array": [{"a": "1", "b": "laptop-47"}, {"a": "1", "b": "desktop-93"}, {"a": "2", "b": "z"}]}