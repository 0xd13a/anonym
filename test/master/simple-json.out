Processing input/simple-json.json
